msctl

```diff
@@ -2186,46 +2326,50 @@ fi
 # pairs to one of the mscs.defaults files mentioned above. Default values in
 # the script will be used unless overridden in one these files.
 #
 # The following keys are available:
-#   mscs-location                - Location of the mscs files.
-#   mscs-worlds-location         - Location of world files.
-#   mscs-versions-url            - URL to download the version_manifest.json file.
-#   mscs-versions-json           - Location of the version_manifest.json file.
-#   mscs-versions-duration       - Duration (in minutes) to keep the version_manifest.json file before updating.
-#   mscs-lockfile-duration       - Duration (in minutes) to keep lock files before removing.
-#   mscs-default-world           - Default world name.
-#   mscs-default-port            - Default Port.
-#   mscs-default-ip              - Default IP address.
-#   mscs-default-version-type    - Default version type (release or snapshot).
-#   mscs-default-client-version  - Default version of the client software.
-#   mscs-default-client-jar      - Default .jar file for the client software.
-#   mscs-default-client-url      - Default download URL for the client software.
-#   mscs-default-client-location - Default location of the client .jar file.
-#   mscs-default-server-version  - Default version of the server software.
-#   mscs-default-jvm-args        - Default arguments for the JVM.
-#   mscs-default-server-jar      - Default .jar file for the server software.
-#   mscs-default-server-url      - Default download URL for the server software.
-#   mscs-default-server-args     - Default arguments for a world server.
-#   mscs-default-initial-memory  - Default initial amount of memory for a world server.
-#   mscs-default-maximum-memory  - Default maximum amount of memory for a world server.
-#   mscs-default-server-location - Default location of the server .jar file.
-#   mscs-default-server-command  - Default command to run for a world server.
-#   mscs-backup-location         - Location to store backup files.
-#   mscs-backup-log              - Lcation of the backup log file.
-#   mscs-backup-excluded-files   - Comma separated list of files and directories excluded from backups.
-#   mscs-backup-duration         - Length in days that backups survive.
-#   mscs-log-duration            - Length in days that logs survive.
-#   mscs-detailed-listing        - Properties to return for detailed listings.
-#   mscs-enable-mirror           - Enable the mirror option by default for worlds (default disabled).
-#   mscs-mirror-path             - Default path for the mirror files.
-#   mscs-overviewer-bin          - Location of Overviewer.
-#   mscs-overviewer-url          - URL for Overviewer.
-#   mscs-maps-location           - Location of Overviewer generated map files.
-#   mscs-maps-url                - URL for accessing Overviewer generated maps.
+#   mscs-location                    - Location of the mscs files.
```

Comment:
This whole section seems to have been changed due to whitespaces changes

Reply:
yes, I increased the white space because the new variable names are long and didn't fit nicely into the column before

Reply:
Ah, I didn't catch that.